app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -219,6 +228,57 @@ ShortFormApplicationController = (
     else if $scope.application.answeredCommunityScreening ==  'Yes'
       ShortFormNavigationService.goToApplicationPage('dahlia.short-form-welcome.overview')
 
+  ########## BEGIN CUSTOM EDUCATOR SCREENING LOGIC ##########
+
+  # reserved for educators
+  $scope.customEducatorIsListing1 = ->
+    $scope.listing.Custom_Listing_Type == 'Educator 1: SFUSD employees only'
+
+  # anyone can apply
+  $scope.customEducatorIsListing2 = ->
+    $scope.listing.Custom_Listing_Type == 'Educator 2: SFUSD employees & public'
+
+  # waitlist only
+  $scope.customEducatorIsListing3 = ->
+    $scope.listing.Custom_Listing_Type == 'Educator 3: Waitlist - SFUSD employees & public'
+
+  $scope.isCustomEducatorListing = ->
+    $scope.customEducatorIsListing1() || $scope.customEducatorIsListing2() || $scope.customEducatorIsListing3()
+
+  $scope.customEducatorIsEducator = ->
+    $scope.application.customEducatorScreeningAnswer == 'Yes'
+
+  $scope.customEducatorIsNotEducator = ->
+    $scope.application.customEducatorScreeningAnswer == 'No'
+
+  $scope.customEducatorCheckScreening = ->
+    $scope.clearEligibilityErrors()
+
+  $scope.customEducatorCapitalizeJobClassifcationNumber = ->
+    form = $scope.form.applicationForm
+    form['customEducatorJobClassificationNumber'].$setViewValue(
+      form['customEducatorJobClassificationNumber'].$viewValue.toUpperCase()
+    )
+
+  $scope.customEducatorValidJobClassificationNumber = (value) ->
+    _.includes(
+      ShortFormHelperService.customEducatorValidJobClassificationNumbers,
+      (value || '').toUpperCase()
+    )
+
+  $scope.customEducatorValidateEligibility = ->
+    if $scope.customEducatorIsListing1() && $scope.customEducatorIsEducator()
+      $scope.clearEligibilityErrors()
+      ShortFormNavigationService.goToApplicationPage('dahlia.short-form-welcome.overview')
+    else if $scope.customEducatorIsListing1() && $scope.customEducatorIsNotEducator()
+      $scope.eligibilityErrors = [$translate.instant('a3_custom_educator_screening.you_must_work_at')]
```

Comment:
The styles causing this are from an existing component. I fixed this by changing the background color to white, because there was not an easy way to darken the hyperlink colors.

Reply:
Looks good!